spec/system/bitklavier_single_item_metadata_spec.rb

```diff
@@ -28,13 +28,14 @@
     expect(page).to have_css '.issue-date-heading'
   end
 
+  # rubocop:disable Layout/LineLength
   it "has expected metadata" do
     visit '/catalog/doi-10-34770-r75s-9j74'
     expect(page).to have_content "bitKlavier Grand Sample Library—Binaural Mic Image"
     author_top_of_page = '<span class="author-name">'
     author_popover_title = 'data-original-title="Trueman, Daniel"'
     author_popover_orcid = 'https://orcid.org/1234-1234-1234-1234'
-    author_popover_affiliation = "<a href='/?f[authors_affiliation_ssim][]=Princeton+Plasma+Physics+Laboratory&amp;q=&amp;search_field=all_fields'>Princeton Plasma Physics Laboratory</a><br/>"
+    author_popover_affiliation = "&lt;a href='/?f[authors_affiliation_ssim][]=Princeton+Plasma+Physics+Laboratory&amp;q=&amp;search_field=all_fields'&gt;Princeton Plasma Physics Laboratory&lt;/a&gt;"
```

Comment:
@kelynch for some reason the encoding has changed and I don't think this was introduced by your PR since this test is also failing in main. I've updated the code to use the proper HTML encoding.

Reply:
Thanks @hectorcorrea!